cli/codegen/generate_code.go

```diff
@@ -0,0 +1,162 @@
+package main
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/apex/log"
+	"github.com/dave/jennifer/jen"
+)
+
+type generateLuaCodeOpts struct {
+	PackageName  string
+	FileName     string
+	PackagePath  string
+	VariablesMap map[string]string
+}
+
+var luaCodeFiles = []generateLuaCodeOpts{
+	generateLuaCodeOpts{
+		PackageName: "admin",
+		FileName:    "cli/admin/lua_code_gen.go",
+		VariablesMap: map[string]string{
+			"adminListFuncBodyTmpl":  "cli/admin/lua/admin_list_func_body_template.lua",
+			"evalFuncGetResBodyTmpl": "cli/admin/lua/eval_func_get_res_body_template.lua",
+			"funcHelpMsgTmpl":        "cli/admin/lua/func_help_msg_template.lua",
+		},
+	},
+	generateLuaCodeOpts{
+		PackageName: "connect",
+		FileName:    "cli/connect/lua_code_gen.go",
+		VariablesMap: map[string]string{
+			"evalFuncBody":           "cli/connect/lua/eval_func_body.lua",
+			"getSuggestionsFuncBody": "cli/connect/lua/suggestions_func_body.lua",
+			"getTitleFuncBody":       "cli/connect/lua/title_func_body.lua",
+		},
+	},
+	generateLuaCodeOpts{
+		PackageName: "connector",
+		FileName:    "cli/connector/lua_code_gen.go",
+		VariablesMap: map[string]string{
+			"callFuncTmpl": "cli/connector/lua/call_func_template.lua",
+			"evalFuncTmpl": "cli/connector/lua/eval_func_template.lua",
```

Comment:
Unlike `funcHelpMsgTmpl`, this templates should be placed here to simplify editing it (it's Lua code).